### PR TITLE
added tests

### PR DESCRIPTION
**Pull request by onedebos**

### CODE REVIEW

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi @onedebos @vieiramanda11 good Job guys, but we have a had a few things 

-  We need to make positive and negative test cases. For example, value == 4, should also have a test for value !== 4
- the game_play file has a player function, please where is the test for that, t seems that wa missed  :smile: 
Please make these changes and submit for another re-review

**Comment by onedebos** [comment]
Added more negative test cases and wrote test for Play method.

File: spec/main_game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+# RSpec.describe '::Game' do
+#   let(:game) { Game.new }
+
+#   describe '#initialize' do
+#     it 'initializes a new game object' do
+#       expect(game.nil?).to eql(false)
+#     end
+#   end
+# end
```
Comment (bolah2009) [inline spec/main_game_spec.rb:13]: #### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

#### Nice job with the testing :ok_hand: 

Few things before this project get a :+1: 

- Test all public methods, examples of methods yet to be tested ~>  `win?`,  `draw?`
   > All the public methods of the classes implemented in project OOP are tested
- Delete commented out code
  > Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.
- Prevent test output from printing game moves, see [here](https://github.com/onedebos/tic-tac-toe-main/pull/4#pullrequestreview-278441480)
- **Is there a reason why only @onedebos is working on this PR?** :thinking: 

Do this and submit for another code review.

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — COMMENTED** [review]
### Current test output using the [`--format`](https://relishapp.com/rspec/rspec-core/v/2-4/docs/command-line/format-option) option

```rspec
$ rspec spec --format documentation

Board
" 1 | 2 | 3 "
"---|---|---"
" 4 | 5 | 6 "
"---|---|---"
" 7 | 8 | 9 "
  has a first row
" 1 | 2 | 3 "
"---|---|---"
" 4 | 5 | 6 "
"---|---|---"
" 7 | 8 | 9 "
  has a first row
" 1 | 2 | 3 "
"---|---|---"
" 4 | 5 | 6 "
"---|---|---"
" 7 | 8 | 9 "
  has a second row
" 1 | 2 | 3 "
"---|---|---"
" 4 | 5 | 6 "
"---|---|---"
" 7 | 8 | 9 "
  has a second row
" 1 | 2 | 3 "
"---|---|---"
" 4 | 5 | 6 "
"---|---|---"
" 7 | 8 | 9 "
  has a third row
" 1 | 2 | 3 "
"---|---|---"
" 4 | 5 | 6 "
"---|---|---"
" 7 | 8 | 9 "
  has a third row
  #is_move_valid?
    accepts a number between 1-9
    rejects any number not between 1-9
  #add_move
    adds move
    rejects invalid move
    updates the state
    doesn't update the state
    #play
      returns true when result == 1
      returns false when result !== 1
    #draw?
      signifies a draw
      only signifies draws

Player
  creates a player object
  receives and sets a player name
  receives and sets a player name
  receives and sets a symbol for the player
  receives and sets a symbol for the player

Finished in 0.00537 seconds (files took 0.07903 seconds to load)
21 examples, 0 failures
```

**Pull request merged**